app/templates/admin/modules.hbs

```diff
@@ -1 +1,33 @@
-{{outlet}}
+<form class="ui form">
+  <div class="ui vertical segment grid">
```

Comment:
make this a basic segment. This page is anyway wrapped in a segment by the parent route.